src/log10/anthropic.py

```diff
@@ -119,9 +119,9 @@ def convert_claude_to_messages(prompt: str):
 
     @staticmethod
     def create_tokens_usage(prompt: str, completion: str):
-        client = anthropic.Anthropic()
-        prompt_tokens = client.count_tokens(prompt)
-        completion_tokens = client.count_tokens(completion)
+        # count_tokens is removed from the API in 0.39.0, consider to deprecate anthropic completion
```

Comment:
anthropic 0.39.0 removed "legacy client.count_tokens() method". We only call create_tokens_usage for message using legacy completion model, like claude-2.1. Tried to use a new API `beta.messages.count_tokens` but got 400 error from the API. 
https://github.com/anthropics/anthropic-sdk-python/releases

Consider the rare usage of legacy models, I set the token to 0 and we could discuss if we want to deprecate the support in future.